byterun/str.c

```diff
@@ -229,6 +242,20 @@ CAMLprim value caml_string_equal(value s1, value s2)
   return Val_true;
 }
 
+CAMLprim value caml_bytes_equal(value s1, value s2)
```

Comment:
What's the point of duplicating the code of these primitives?  I understand it is useful to call different primitives (so that e.g. js_of_ocaml can plug different alternatives), but why duplicate the function bodies in the C runtime?